docs/src/main/asciidoc/deploying-to-openshift-s2i.adoc

```diff
@@ -126,7 +126,7 @@ We are going to create an OpenShift `build` executing it:
 [source,shell, subs="attributes"]
 ----
 # To build the image on OpenShift
-oc new-app registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift~{quickstarts-clone-url} --context-dir=getting-started --name=quarkus-quickstart
+oc new-app registry.access.redhat.com/ubi8/openjdk-8:latest~{quickstarts-clone-url} --context-dir=getting-started --name=quarkus-quickstart
```

Comment:
Do we have a JDK 11 image? Because we are actively pushing JDK 11 now, this one looks like an oversight.

Reply:
Yes, it's `ubi8/openjdk-11:latest`. If you'd prefer that in the PR I'll edit it and test accordingly.

Reply:
tested; Updated to use 11; added signed-off-by; wrapped the commit message (GitHub UI doesn't do that)